### DISCUSSION TITLE
Hello !  Question about version chaos

### DISCUSSION THREAD

**Discussion by supertick** [serialport/node-serialport#2819] [General]
Hello !
First .. thanks for a great project and all the contributions :) 
I've surfed around a bit and see Electron + serialport challenges.   It doesn't surprise me, maintaining binaries over multiple platforms can be a beast.
I saw the  https://github.com/serialport/electron-serialport  project, and tried it.  It works, but the electron version of that is still way behind (so is the serialport version)

I am currently running electron 29.1.4 and attempting to run serialport 12.0.0
NODE_MODULE_VERSION 120. This version of Node.js requires
[1] NODE_MODULE_VERSION 121. Please try re-compiling or re-installing
[1] the module (for instance, using `npm rebuild` or `npm install`).

I tried installing electron-rebuild as far as I'm aware of ... the error stays the same.

In short, I'm just trying to get some guidance as to the best strategy to make electron+node+serialport happy 
I was hoping not to downgrade electron as I just spent considerable time getting it to 29.1.4 

Thanks

**Comment by supertick** [comment]
Heh, nevermind.
I have to run npx rebuild not just install it ... duh :facepalm:
